haystack/nodes/reader/base.py

```diff
@@ -122,7 +122,9 @@ def run(self, query: str, documents: List[Document], top_k: Optional[int] = None
             # filters out empty documents
             relevant_documents = list(
                 {
-                    label.document.id: label.document for label in labels.labels if label.document.content.strip() != ""
```

Comment:
@ju-gu From what I understand using this approach, this means that for TableQA `relevenat_documents` will always be `0`, since we are skipping the counting of all documents whose content is not of type string. I believe this means the retrieval metrics for TableQA will be incorrect. Do you agree with this assessment?